cmd/juju/cloud/list.go

```diff
@@ -302,7 +360,7 @@ func formatCloudsTabular(writer io.Writer, value interface{}) error {
 		return cloudNames
 	}
 
-	printClouds := func(someClouds map[string]*CloudDetails, color *ansiterm.Context) {
```

Comment:
The colour legend is not shown but for me it still adds significant value as it breaks up the wall of text. And it's not that difficult to discern looking at the output what the breakdown is. YMMV. DId we get agreement to remove the colour?

Reply:
Color had no meaning for users beforehand (only devs that saw the code knew that there is a difference between public, built-in and personal clouds). Now with clearer division between client and controller clouds, it makes no sense.

Reply:
The other bits was that there were 3 blocks displayed, each sorted alphabetically, but again that grouping made no sense. All client clouds are now displayed together and sorted together.

Reply:
In which case my comment is totally irrelevant,